test/sequential/xyk-pallet.api.maintenance-config.test.ts

```diff
@@ -264,3 +274,75 @@ async function getSudoError(
 
   expect(sudoEventError.name).toEqual(expectedError);
 }
+
+test("maintenance- bootstrap can be run in maintenanceMode, but you can't provide it and the pool will not be created", async () => {
+  await checkMaintenanceStatus(false, false);
+  await checkLastBootstrapFinalized(sudo);
+  const bootstrapCurrency = await createNewBootstrapCurrency(sudo);
+  const waitingPeriod = 10;
+  const bootstrapPeriod = 8;
+
+  [testUser1] = setupUsers();
+
+  await setupBootstrapTokensBalance(bootstrapCurrency, sudo, [testUser1]);
+
+  await sudo.mint(bootstrapCurrency, testUser1, toBN("1", 20));
```

Comment:
reuse the Asset.mint and add it to the next batchAsSudoFinalize.  One day we will get rid of this sudo.mint stuff.

Reply:
Done